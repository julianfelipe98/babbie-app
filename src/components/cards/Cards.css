hr {
  width: 50%;
  height: 2px;
  background-color: #e4e4e4;
  border: none;
}
#text-info {
  color: #222;
  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
  "Lucida Sans", Arial, sans-serif;
}
.main-title{
    padding-bottom: 5%;
    text-align: center;
}