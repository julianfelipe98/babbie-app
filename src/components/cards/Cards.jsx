import React from "react";
import { CardInfo } from "./card/CardInfo";
import "./Cards.css";
export const Cards = () => {
  const cardsArray = [
    {
      title: "Crea tu cuenta",
      subtitle: "Crea o ingresa tu cuenta para comenzar.",
      paragraph:
        "Registra a todas tus mascotas en tu cuenta pra llevarlas siempre contigo.",
    },
    {
      title: "Explora el historial medico",
      subtitle: "Registra y consulta historiales medicos.",
      paragraph:
        "Puedes registrar cualquier evento de tus mascotas y no perderlos de vista.",
    },
    {
      title: "Siempre contigo",
      subtitle: "Todo el historial medico a tus manos.",
      paragraph:
        "Consulta el historial medico de tus mascotas en cualquier momento y lugar.",
      btn: "true",
    },
  ];
  return (
    <div className="cards">
    <div className="main-title">
      <hr className="solid"></hr>
      <h1 id="text-info">Â¿Como funciona?</h1>
    </div>
      {cardsArray.map((card, i) => (
        <CardInfo
          key={i}
          id={i}
          title={card.title}
          subtitle={card.subtitle}
          paragraph={card.paragraph}
          btn={card.btn}
        />
      ))}
    </div>
  );
};
