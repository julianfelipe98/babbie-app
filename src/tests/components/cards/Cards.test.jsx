import { shallow } from "enzyme";
import { Cards } from "../../../components/cards/Cards";

describe("Cards component tests", () => {
  test("should render <Cards/> component in the correct way", () => {
    const wrapper = shallow(<Cards />);
    expect(wrapper).toMatchSnapshot();
  });
});
