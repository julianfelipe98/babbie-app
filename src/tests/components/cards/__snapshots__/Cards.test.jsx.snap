// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards component tests should render <Cards/> component in the correct way 1`] = `
<div
  className="cards"
>
  <div
    className="main-title"
  >
    <hr
      className="solid"
    />
    <h1
      id="text-info"
    >
      Â¿Como funciona?
    </h1>
  </div>
  <CardInfo
    id={0}
    key="0"
    paragraph="Registra a todas tus mascotas en tu cuenta pra llevarlas siempre contigo."
    subtitle="Crea o ingresa tu cuenta para comenzar."
    title="Crea tu cuenta"
  />
  <CardInfo
    id={1}
    key="1"
    paragraph="Puedes registrar cualquier evento de tus mascotas y no perderlos de vista."
    subtitle="Registra y consulta historiales medicos."
    title="Explora el historial medico"
  />
  <CardInfo
    btn="true"
    id={2}
    key="2"
    paragraph="Consulta el historial medico de tus mascotas en cualquier momento y lugar."
    subtitle="Todo el historial medico a tus manos."
    title="Siempre contigo"
  />
</div>
`;
